prql-compiler/src/test.rs

```diff
@@ -3320,3 +3320,21 @@ fn test_params() {
     "###
     );
 }
+
+#[test]
```

Comment:
Are we OK to include a reference to the issue?

```suggestion
// TODO: fix this based on https://github.com/PRQL/prql/pull/1818
#[test]
```